configuration/enable-offline-mode.adoc

```diff
@@ -46,14 +46,17 @@ include::modules/enable-offline-mode-roxctl.adoc[leveloffset=+2]
 [id="update-scanner-definitions-in-offline-mode"]
 == Updating Scanner definitions in offline mode
 
-Scanner contains a local vulnerability definitions database. When {product-title} runs in normal mode (connected to the internet), Scanner fetches new vulnerability definitions from the internet and updates its database.
+Scanner contains a local vulnerability definitions database. If {rh-rhacs-first} is running in normal mode, Scanner fetches new vulnerability definitions from Central. Central retrieves the updates from the internet, and then updates the Central database.
 //TODO: Add link to Vulnerability definitions
 
-However, when you are using {product-title} in offline mode, you must manually update Scanner definitions by uploading them to Central.
+However, if you are using {product-title-short} in offline mode, you must manually update the definitions. 
+By uploading the definitions to Central, you can manually update the definitions.
 
-When {product-title} runs in offline mode, Scanner checks for new definitions from Central. If new definitions are available, Scanner downloads the new definitions from Central, marks them as default, and then uses the updated definitions for scanning images.
+If the product is running in offline mode, Scanner checks with Central for new data approximately every 5 minutes. Central also checks for new data approximately every 5 minutes. 
 
-To update the definitions in offline mode:
+The data is updated approximately every 3 hours. If new definitions are available, Scanner downloads them from Central, marks them as default and then uses the updated definitions to scan images.
```

Comment:
I think it makes more sense to talk about the frequency of the data's update cycle after we mention the user has to download the data, which is done below. At the moment, it's not very clear which data we are talking about.

> marks them as default

I don't know what this means

```suggestion
The offline data source is updated approximately every 3 hours. Once uploaded to Central, Scanner will download it and update its local vulnerability database.
```